ops.py

```diff
@@ -150,3 +150,26 @@ def forward(self, x):
             if device:
                 self.to(device)
             return x
+
+    class Conv2d(GGMLLayer):
+        comfy_cast_weights = True
+
+        def __init__(self, *args, device=None, dtype=None, **kwargs):
+            super().__init__(device=device, dtype=dtype)
+            _ = kwargs.pop("kernel_size", None)
+            self.kwargs=kwargs
+
+        def forward(self, x):
+            # lowvram hack

```

Comment:
I can deduplicate this part and clean it up once we merge it, I had it originally when I was adding T5 but ended up not needing it.

Reply:
you mean how the conv thing duplicates the linear layer code? yeah, definitely, was just to make sure it actually worked. there should be a better way to do it. it seemed weird/possibly wrong how it was inheriting from the normal linear layer.